adafruit_emc2101/emc2101_regs.py

```diff
@@ -0,0 +1,81 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022 Ruth Ivimey-Cook for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+"""
+`adafruit_emc2101.emc2101_regs`
+================================================================================
+
+Brushless fan controller EMC2101 Register addresses.
+
+* Author(s): Bryan Siepert, Ruth Ivimey-Cook
+
+Implementation Notes
+--------------------
+
+**Hardware:**
+
+* `Adafruit EMC2101 Breakout
+  <https://adafruit.com/product/4808>`_ (Product ID: 4808)
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://circuitpython.org/downloads
+
+* Adafruit's Bus Device library:
+  https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
+
+* Adafruit's Register library:
+  https://github.com/adafruit/Adafruit_CircuitPython_Register
+
+"""
+
+
+from micropython import const
+
+#
+# EMC2101 Register Addresses
+#
+class EMC2101_Regs:  # pylint: disable=too-few-public-methods
```

Comment:
We could probably avoid having to use a class if we change how we're accessing the registers in the other files:

```python
from micropython import const

INTERNAL_TEMP = const(0x00)
EXTERNAL_TEMP_MSB = const(0x01)
...
```

Then  we can access these in the other files like this, while still protecting the namespace:

```python
from adafruit_emc2101 import emc2101_regs

something = emc2101_regs.INTERNAL_TEMP
```

That should resolve that `pylint` warning about too few public methods too!

Reply:
I'd say so, I don't like using `class`es unless there's methods and things that are needed, which is what that `pylint` warning is saying.  But I agree that breaking it into it's own file (or namespace) is a good idea, so that's a good balance of organization and memory.